ccd-definition/ComplexTypes/CareSupervision/RespondentNew-nonprod.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "RespondentNew",
+    "ListElementCode": "legalRepresentation",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Do they have legal representation?",
+    "SecurityClassification": "Public"
```

Comment:
In the prototype I see a label "Legal representation" should be before this radio button, unless design has changed